### PR TITLE
[WIP]Add e2e test for streaming

### PR DESCRIPTION
**Pull request by heartlock**

### CODE REVIEW

File: test/e2e/streaming.go [hunk 1]
```diff
@@ -17,136 +17,311 @@ limitations under the License.
...
 	)
 
 	BeforeEach(func() {
-		runtimeClient = f.Client.FraktiRuntimeService
-		imageClient = f.Client.FraktiImageService
+		rc = f.Client.FraktiRuntimeService
+		ic = f.Client.FraktiImageService
 	})
 
-	It("test exec a command in container synchronously and successfully", func() {
```
Comment (Crazykev) [inline test/e2e/streaming.go]: This is a `ExecSync` test, why remove this?

File: test/e2e/streaming.go [hunk 2]
```diff
@@ -17,136 +17,311 @@ limitations under the License.
...
+		})
+
+		AfterEach(func() {
+			By("stop PodSandbox")
+			rc.StopPodSandbox(podID)
+			By("delete PodSandbox")
+			rc.RemovePodSandbox(podID)
+		})
 
-	It("test exec a command in container synchronously and failed", func() {
```
Comment (Crazykev) [inline test/e2e/streaming.go]: Also a `ExecSync` test

File: test/e2e/streaming.go [hunk 3]
```diff
@@ -17,136 +17,311 @@ limitations under the License.
...
+	framework.ExpectNoError(err, "Failed to create container: %v", err)
 
-	return podId, containerId
+	By("start container")
+	err = rc.StartContainer(containerID)
+	framework.ExpectNoError(err, "Failed to start container: %v", err)
+
+	// sleep 2s to make sure container start is ready, workaround for random failed in travis
+	// TODO: remove this
+	time.Sleep(2 * time.Second)
```
Comment (Crazykev) [inline test/e2e/streaming.go]: This is not needed anymore, you can remove this and above one.

File: test/e2e/streaming.go (hunk 2 above)
Comment (heartlock) [inline test/e2e/streaming.go]: I want to put it to other place @Crazykev

Comment (Crazykev) [inline test/e2e/streaming.go]: @heartlock Sure, that's fine to me.

**Comment by feiskyer** [comment]
P1 items of critest have been done. I'd prefer to use critest for our integration tests.

**Comment by feiskyer** [comment]
@heartlock  Would you like to change the integration test to critest?

**Comment by heartlock** [comment]
Replace Frakti e2e test with critest? It's a good idea! @feiskyer

**Comment by heartlock** [comment]
close this, open another for changing  the integration test to critest

**Comment by Crazykev** [comment]
@feiskyer Do we still need some frakti specified test case? such as networking and multi-runtime related?

**Comment by feiskyer** [comment]
@Crazykev Yep, of course.

**Pull request closed without merging**